internal/api/resolver_query_scraper.go

```diff
@@ -113,7 +123,52 @@ func (r *queryResolver) ScrapeScene(ctx context.Context, scraperID string, scene
 		return nil, err
 	}
 
-	return marshalScrapedScene(content)
+	ret, err := marshalScrapedScene(content)
+	if err != nil {
+		return nil, err
+	}
+
+	filterSceneTags([]*models.ScrapedScene{ret})
+
+	return ret, nil
+}
+
+// filterSceneTags removes tags matching excluded tag patterns from the provided scraped scenes
+func filterSceneTags(scenes []*models.ScrapedScene) {
+	excludePatterns := manager.GetInstance().Config.GetScraperExcludeTagPatterns()
+	var excludeRegexps []*regexp.Regexp
+
+	for _, excludePattern := range excludePatterns {
+		reg, err := regexp.Compile(strings.ToLower(excludePattern))
+		if err != nil {
+			logger.Errorf("Invalid tag exclusion pattern: %v", err)
+		} else {
+			excludeRegexps = append(excludeRegexps, reg)
+		}
+	}
+
+	if len(excludeRegexps) == 0 {
+		return
+	}
+
+	for _, s := range scenes {
+		var newTags []*models.ScrapedTag
+		for _, t := range s.Tags {
+			ignore := false
+			for _, reg := range excludeRegexps {
+				if reg.MatchString(strings.ToLower(t.Name)) {
+					ignore = true
```

Comment:
In the removed code the ignored tags (t.Name) are also added to a list and then log printed. Should we readd that code?